src/views/catalog.vue

```diff
@@ -20,8 +20,8 @@
       </ion-toolbar>
     </ion-header>
 
-    <ion-content :fullscreen="true">
-      <div v-if="segment == 'all'">
+    <!-- Segments -->
+    <div v-if="segment == 'all'">
```

Comment:
I meant that we should remove this condition here because the entire content isn't tied to the segments they are going to be used for filters inside the results.

Reply:
Ok sir

Reply:
Done sir 
![segment](https://user-images.githubusercontent.com/82817616/152475349-a037962c-d140-44b1-89c6-2444b3a069a3.png)